examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+# EKS Cluster with Amazon Managed Grafana and Amazon OpenSearch Service
+
+This example demonstrates how to use the AWS EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Prometheus for metrics and Fluent Bit for logging. It also provisions Amazon Managed Prometheus (AMP), Amazon OpenSearch Service within a VPC, and integrates AMP with Amazon Managed Grafana (AMG). It also deploys an EC2 instance to let us test OpenSearch. Lastly, it includes a sample workload, provisioned with ArgoCD, to generate logs and metrics.
+
+Prometheus sends application metrics to AMP, and AMG ingests data from AMP. Fluent Bit sends logs to Amazon OpenSearch Service.
+
+---
+**NOTE**
+
+For the sake of simplicity in this example, we store sensitive information and credentials in `dev.tfvars`. This should not be done in a production environment. Instead, use an external secret store such as AWS Secrets Manager and use the [aws_secretsmanager_secret](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/secretsmanager_secret) data source to retrieve them.
+
+---
+
+## How to Deploy
+
+### Prerequisites
```

Comment:
Lets add all the pre-requisites for deploying this example.
1. Terrafrom
2. AWS Account
3. Kubectl etc.